### ISSUE TITLE
Explicit import in Examples

### ISSUE THREAD

**Issue by grosenkj** [simpeg/simpeg#235]
We should import modulus explicitly in Examples. Not do:
```python
from SimPEG import *
```
Instead, either do 
```python
from SimPEG import [specific Modulus]
```
or 
```python
import SimPEG [as simpeg]
```
to make the origin of the objects clearer.

**Comment by lheagy** [comment]
that is a good call, show which module we are using (or at least include a comment of what is imported on * )

**Comment by lheagy** [comment]
it starts to get pretty long... should we add a description in the docs of what is imported on * instead?

**Comment by rowanc1** [comment]
This has been fixed in #471
